src/app/clusters/level-control/level-control.cpp

```diff
@@ -139,6 +146,33 @@ static void timerCallback(System::Layer *, void * callbackContext)
     emberAfLevelControlClusterServerTickCallback(static_cast<EndpointId>(reinterpret_cast<uintptr_t>(callbackContext)));
 }
 
+static uint32_t calculateNextWaitTimeMs(NextWaitTimeState * nextwaitTimeState, uint32_t delayMs)
```

Comment:
Should nextwaitTimeState be a reference, since it can't be null?  And maybe named `nextWaitTimeState`?

Reply:
Done